### PR TITLE
Slideshow images

### PR DESCRIPTION
**Pull request by megalobamboozlementarianism**

### CODE REVIEW

File: models.js [hunk 1]
```diff
@@ -21,10 +21,14 @@ const imageSchema = new Schema({
     type: String,
     required: true,
   },
-  caption: {
+  keyword: {
     type: String,
     required: true,
   },
+  savedPlace: {
```
Comment (ed42311) [inline models.js]: Is this confusing in terms of how the user is interacting with the page?

Comment (ceotter) [inline models.js]: Make this name more descriptive.

File: server.js [hunk 2]
```diff
@@ -65,6 +65,16 @@ app.use(bodyParser.urlencoded({ extended: true }));
 // parse the response body back into a json object
 app.use(bodyParser.json());
 
+app.use((req, res, next)=>{
```
Comment (ceotter) [inline server.js]: Shouldn't be run in production. Delete if you want.

**Review by ceotter — CHANGES_REQUESTED** [review]
See above comments.

**Review by ceotter — APPROVED** [review]

**Pull request merged**